test/contracts/AuctionRegistrar.cpp

```diff
@@ -239,27 +239,27 @@ class AuctionRegistrarTestFramework: public SolidityExecutionFramework
 		{
 			callString("reserve", _name);
 		}
-		u160 owner(string const& _name)
+		h160 owner(string const& _name)
```

Comment:
Why not `Address`?

Reply:
https://github.com/ethereum/solidity/pull/10447#issuecomment-736596814